Sources/Model/TransferMethod/HyperwalletPrepaidCard.swift

```diff
@@ -32,6 +32,11 @@ public final class HyperwalletPrepaidCard: HyperwalletTransferMethod {
         return getField(TransferMethodField.cardType.rawValue)
     }
 
+    /// The prepaid card's cvv
+    public var cvv: String? {
```

Comment:
Where did you see this property for prepaid card?

Reply:
We shouldn't expose cvv. API should never return cvv in the response so kindly remove it